research/src/fetch_campaign_donations.py

```diff
@@ -0,0 +1,346 @@
+import pandas as pd
+import urllib.request
+import os        
+import sys
+import zipfile
+import time
```

Comment:
Would you mind following [PEP8 style for imports](https://www.python.org/dev/peps/pep-0008/#imports)? First we'd have native modules, then 3rd party, and then local modules, each block separated by a blank line.

Reply:
I am not really sure if now I am following PEP8 in the imports... could you take a look again?